gaitutils/videos.py

```diff
@@ -46,21 +47,49 @@ def convert_videos(vidfiles, check_only=False):
         # return True if all conversion targets already exists
         return all(p.is_file() for p in convfiles.values())
 
-    # XXX: this disables Windows protection fault dialogs
-    # needed since ffmpeg2theora may crash after conversion is complete (?)
-    SEM_NOGPFAULTERRORBOX = 0x0002  # From MSDN
-    ctypes.windll.kernel32.SetErrorMode(SEM_NOGPFAULTERRORBOX)
+    if platform.system() == 'Windows':
+        # XXX: this disables Windows protection fault dialogs
+        # needed since ffmpeg2theora may crash after conversion is complete (?)
+        SEM_NOGPFAULTERRORBOX = 0x0002  # From MSDN
```

Comment:
This is bit of a hack and might lead to confusion when protection fault dialogs are not shown, not sure if it's needed any more. Ideally it should be tested if ffmeg2theora still behaves like this.